packages/libs/deer/rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+# Please also update the badges in `README.md`, `src/lib.rs`, and `macros/`
+channel = "nightly-2022-09-27"
+components = ['rust-src', 'clippy', 'rustfmt']
```

Comment:
We should either add `miri` or disable `miri` in the Makefile and forbid unsafe code.

Reply:
fixed in fixed in [bb95c16](https://github.com/hashintel/hash/pull/1163/commits/bb95c166353807767082715d378f75bad27392b7), Makefile deferred until CI PR